core/validation/specific_validations/towns_fund_round_four.py

```diff
@@ -49,9 +49,24 @@ def validate_project_risks(workbook: dict[str, pd.DataFrame]) -> list["TownsFund
         ]
 
 
-def validate_programme_risks(workbook: dict[str, pd.DataFrame]) -> list["TownsFundRoundFourValidationFailure"]:
-    # TODO: validate there is a risk for each project.
-    return []
+def validate_programme_risks(workbook: dict[str, pd.DataFrame]) -> list["TownsFundRoundFourValidationFailure"] | None:
+    """Validates that each submission has at least one Programme level Risk.
```

Comment:
this docstring description doesn't match the code (logic condition or failure message)  wrt number of programme risks required